triple-reads-backend/public/testFiles/updateBookTest.ts

```diff
@@ -0,0 +1,59 @@
+const bookFormUpdate = document.getElementById('bookFormUpdate') as HTMLFormElement;
+bookFormUpdate.addEventListener('submit', async (event) => {
+    event.preventDefault();
+
+    const formData = new FormData(bookFormUpdate);
+
+    const authorsInput = formData.get('authors');
+    const authorNames = authorsInput ? (authorsInput as string).split(',').map(author => author.trim()) : [];
+
+    const genresInput = formData.get('genres');
+    const genreNames = genresInput ? (genresInput as string).split(',').map(genre => genre.trim()) : [];
+
+    const requestBody = {
+        title: formData.get('title'),
+        isbn: formData.get('isbn'),
```

Comment:
Are we going to user an option to edit the isbn number?
If this is the case, then we should check if the isbn already exists.
However, I think there should not be an option to edit the isbn...